tutorial/sample-problems/Algebra/GraphToolCustomChecker.pg

```diff
@@ -159,10 +159,10 @@ The equation of the circle given by the equation
 
     [`[$circle_eq_lhs] = [$r ** 2]`]
 
-has center at [`([$h],[$k])`] and radius [$r].  To enter the graph, click the
-circle tool, then click on the center at [`([$h],[$k])`] and then click on a
-second point that is [$r] units from the center.  This is easiest going left,
-right, up or down from the center.
+has center at [`([$h],[$k])`] and radius [$r]. To enter the graph,
+click the circle tool, then click on the center at [`([$h],[$k])`]
+and then click on a second point that is [$r] units from the center.
+This is easiest going left, right, up or down from the center.
```

Comment:
Why did you reformat this?  This already fit nicely into an 80 character limit (which is consistent with the default perltidy character width).  So there is no reason for this reformatting to be done.

Reply:
Not sure?  reverted it.